keras/backend/tensorflow_backend.py

```diff
@@ -432,6 +436,10 @@ def switch(condition, then_expression, else_expression):
     return tf.python.control_flow_ops.cond(condition,
                                            lambda: then_expression,
                                            lambda: else_expression)
+def broadcasting_select(condition, A, B):
```

Comment:
OK I've restructured that a bit. Though it needs something more major since it still doesn't work in TF. I'm looking into that today.